src/green/InterfacesImpl.hpp

```diff
@@ -111,17 +111,16 @@ class LnTransformedRadial {
    *  \param[out] drhodr state vector holding the first and second derivative
    *  \param[in] r position on the integration grid
    */
-  void operator()(const StateType & rho, StateType & drhodr, const double r) {
+  void operator()(const StateType & rho, StateType & drhodr, const double y) {
     // Evaluate the dielectric profile
     double eps = 0.0, epsPrime = 0.0;
-    pcm::tie(eps, epsPrime) = eval_(r);
+    pcm::tie(eps, epsPrime) = eval_(exp(y));
     if (utils::numericalZero(eps))
```

Comment:
I don't think `eps` can ever be zero here. Plus shame on me for using `throw` instead of `PCMSOLVER_ERROR`. Can you fix that?